src/main/resources/default/templates/storage/object.html.pasta

```diff
@@ -63,6 +63,13 @@
             <p></p>
         </div>
         <div class="col-md-9">
+            <i:if test="@bucket.getName() == 'media' || bucket.getName() == 'public'">
```

Comment:
make configurable. biz does not know the media and public bucket. config per bucket